app/controllers/feedback_controller.rb

```diff
@@ -0,0 +1,25 @@
+class FeedbackController < ApplicationController
+
+	# GET /feedback
```

Comment:
Remove all such comments please. It is known fact that index is used for GET in rails

Reply:
got rid of them now